playbook/app/pb_kits/playbook/pb_user/_user.tsx

```diff
@@ -20,11 +22,13 @@ type UserProps = {
   htmlOptions?: {[key: string]: string | number | boolean | (() => void)},
   id?: string,
   name?: string,
+  nameTypeKit?: "title" | "body" | "caption" | "detail"
```

Comment:
Code looks great @gmorris1242 , small feedback on naming here. Can we rename this prop to `nameStyle`? That would be more in keeping with our system